### DISCUSSION TITLE
Unabled to register again on Github Edcuation

### DISCUSSION THREAD

**Discussion by theoribbi** [community/community#128377] [GitHub Education]
### Hi everyone,

Hello,

I send this morning a request to be part of the github student program. 
I received an email this afternoon that say my billing information is missing, so i fill my information, logout, and after login like the instruction said.
But when i try to send out a new request, it's impossible, i check on the request menu and it's a 413 error "Payload too large".

I already resize my image, i try with my phone too but impossible

Can you help me please ?

Great regards

Théo
![image](https://github.com/community/community/assets/62778864/c0df4665-369b-4509-b903-2ea8f942fd2b)

**Comment by iamdulanga** [comment]
### Hello @Theo2903 

> 1. Resize Your Image Further:

- Even if you've already resize your image, it might still be too large. Try reducing the file size even more using an image compression tool like TinyPNG or JPEG-Optimizer.
- Ensure the image is in a commonly accepted format such as JPEG or PNG.

> 2. Clear Browser Cache:

- Sometimes, clearing your browser cache can help resolve issues related to large payload errors. You can clear your cache in the browser settings.

> 3. Use a Different Browser:

- Try using a different web browser or an incognito/private browsing window to submit your request.

I hope you will fix your problem.. after try these solutions..
